packages/cfd/src/Containers/cfd-compare-accounts/cfd-compare-accounts-description.tsx

```diff
@@ -30,7 +35,21 @@ const CFDCompareAccountsDescription = ({ trading_platforms, is_demo }: TCompareA
             {selected_region === REGION.NON_EU && (
                 <div className='compare-cfd-account-text-container__separator'>
                     <Text as='h1' weight='bold' size='m' align='center'>
-                        {juridisction_data.spread}
+                        <div className='compare-cfd-account-spread'>
+                            {juridisction_data.spread}
+                            {market_type_shortcode === 'all_zero_spread_bvi' && (
```

Comment:
Better not to hardcode this if you've defined it in the cfd-config `ALL_ZERO_SPREAD_BVI`